jbmc/regression/jbmc/class-literals/Test.java

```diff
@@ -0,0 +1,13 @@
+
+public class Test {
+
+  public static void main() {
+
+    assert ExampleAnnotation.class.isAnnotation();
```

Comment:
Possibly being dense but there is no `.desc` file in this PR - shouldn't there be?

Reply:
Er, yes. Will fix.